__tests__/PropTypesDevelopmentStandalone-test.js

```diff
@@ -486,6 +486,77 @@ describe('PropTypesDevelopmentStandalone', () => {
     });
   });
 
+  describe('ElementType Types', () => {
+    it('sould support native component', () => {
```

Comment:
`s/sould/should/g` (but it's even better when tests don't use the word "should" - ie, `it supports native components`)